content/en/docs/tasks/administer-cluster/memory-manager.md

```diff
@@ -46,7 +46,7 @@ Preceding v1.22, the `kubelet` must be started with the following flag:
 
 in order to enable the Memory Manager feature.
 
-## How Memory Manager Operates?
+## How does the Memory Manager Operates?
```

Comment:
?

Reply:
IMO we can keep this line as is prior to the PR

Reply:
Or it can be:
```suggestion
## How does the Memory Manager Operate?
```

Reply:
I took the opportunity to improve the grammar here but happy to remove it if its to much noise.